### ISSUE TITLE
Vertical scroll bar doesn't appear.

### ISSUE THREAD

**Issue by Surakshaajith** [skydoves/PowerSpinner#21]
- Library Version: 1.0.8 (latest)
- Affected Device: Doesn't work on any device 
 
I am trying to add vertical scrollbar to the dropdown but it doesn't appear. I tried applying vertical scrollbar in xml as well as in Java. 
I am restricting the dropdown height so that the scrollbar appears but it doesn't work
```

android:maxHeight="100dp"  
android:fadeScrollbars="false"
android:scrollbarFadeDuration="0" 
android:verticalScrollbarPosition="right" 
android:scrollbarThumbVertical="@android:color/black"
android:scrollbarAlwaysDrawVerticalTrack="true" 
android:scrollbarSize="20dp" 
android:scrollbarStyle="outsideInset" 
android:scrollbars="vertical"


I also tried changing this in java but scrollbar doesn't work. 
Please help me solve this. Thanks

**Comment by skydoves** [comment]
Hi, @Surakshaajith
I will add a `scrollbar` to the popup in the next version, but we can't change the color, width of the `scrollbar` because of the RecyclerView's API is limited.
Thank you for your issue :)

![screenshot1589169265](https://user-images.githubusercontent.com/24237865/83139049-ba9e7f00-a126-11ea-9160-09d19f4cf33b.png)

**Comment by skydoves** [comment]
Released a new version [1.0.9](https://github.com/skydoves/PowerSpinner/releases/tag/1.0.9).
Thank you for your issue :)
